INSTALL.md

```diff
@@ -202,6 +202,77 @@ Possible sources of errors:
   and [compileBLAS.cmd](https://github.com/AMICI-dev/AMICI/blob/master/scripts/compileBLAS.cmd)
   may serve as guidance on how to install openBLAS using msvc.
 
+### Windows using MSVC (Visual Studio)
+
+#### Visual Studio
+We assume that Visual Studio (not to be confused with Visual Studio Code) is already installed. Using Visual Studio Installer, the following components need to be included:
+
+* Microsoft Visual C++ (MSVC). This is part of multiple packages, including Desktop Development with C++.
+* Windows Universal C Runtime. This is an individual component and installs some DLLs that we need.
+
+#### openBLAS
+To install open BLAS, download the following scripts from AMICI:
+
+https://github.com/ICB-DCM/AMICI/blob/master/scripts/installOpenBLAS.ps1
+https://github.com/ICB-DCM/AMICI/blob/master/scripts/compileBLAS.cmd
+
+The first script needs to be called in Powershell, and it needs to call compileBLAS.cmd, so you will need to modify line 11:
+
+    `C:\Users\travis\build\AMICI\scripts\compileBLAS.cmd`
+
+so that it matches your directory structure.
+This will download openBLAS and compile it, creating
+
+    `C:\BLAS\lib\openblas.lib`
```

Comment:
Regarding formatting: Either indentation or backticks for code formatting. You can use any decent markdown editor with preview function, or check https://github.com/AMICI-dev/AMICI/blob/thomassligon/INSTALL.md#python to see how it looks.